ocean_lib/services/v4/service.py

```diff
@@ -0,0 +1,109 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""
+    Service Class for V4
+    To handle service items in a DDO record
+"""
+from typing import Dict, Any, Optional
+
+from enforce_typing import enforce_types
+
+from ocean_lib.common.agreements.service_types import ServiceTypesV4, ServiceTypesNames
+
+
+@enforce_types
+class NFTService:
+    """Service class to create validate service in a V4 DDO."""
+
+    SERVICE_TYPE = "type"
```

Comment:
I would ditch these constants. The usage seems limited and they seem to impair readability.